web/ce/store/pages/extended-base-page.ts

```
@@ -0,0 +1,16 @@
+import { TPage, TPageExtended } from "@plane/types";
+import { RootStore } from "@/plane-web/store/root.store";
+import { TBasePageServices } from "@/store/pages/base-page";
+
+export type TExtendedPageInstance = TPageExtended & {
+  asJSONExtended: TPageExtended;
+};
+
+export class ExtendedBasePage implements TExtendedPageInstance {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  constructor(store: RootStore, page: TPage, services: TBasePageServices) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)